backend/server/controllers/questions.js

```diff
@@ -0,0 +1,136 @@
+const { Question } = require('../models');
+
+module.exports = {
+  create(req, res) {
+    // check that params are not null, undefined or empty string
+    if (!req.body.text) {
+      return res.status(400).send({
+        status: 400,
+        message: 'The attribute "text" of an instance of "Question" cannot be empty.',
+      });
+    }
+    if (!req.body.initialImage) {
+      return res.status(400).send({
+        status: 400,
+        message: 'The attribute "initialImage" of an instance of "Question" cannot be empty.',
+      });
+    }
+    if (!req.body.finalImage) {
+      return res.status(400).send({
+        status: 400,
+        message: 'The attribute "finalImage" of an instance of "Question" cannot be empty.',
+      });
+    }
+    if (!req.body.initialSound) {
+      return res.status(400).send({
+        status: 400,
+        message: 'The attribute "inititalSound" of an instance of "Question" cannot be empty.',
+      });
+    }
+    if (!req.body.finalSound) {
+      return res.status(400).send({
+        status: 400,
+        message: 'The attribute "finalSound" of an instance of "Question" cannot be empty.',
+      });
+    }
+
+    return Question
+      .create({
+        text: req.body.text,
+        initialImage: req.body.initialImage,
+        finalImage: req.body.finalImage,
+        initialSound: req.body.initialSound,
+        finalSound: req.body.finalSound,
+        user_id: req.body.user_id,
```

Comment:
Each question should belong to an experiment, not a user. Please, fix this, @maverick2000. Add also a check for the experiment_id before attempting to create the question. Thanks.